packages/nx-plugin/mock/utils/executor.ts

```diff
@@ -0,0 +1,16 @@
+import { NormalizedExecutorContext } from '../../src/executors/internal/context';
+
+export function normalizedExecutorContext(
+  nameOrOpt: string | { projectName: string } = 'my-lib',
```

Comment:
What's the sense in having a default value? I don't think that assuming there's a project called `my-lib` is likely to create a working configuration.

Reply:
Would also be more clear to read the test if I remove the default value.

Reply:
removed default value